mypyc/lib-rt/str_ops.c

```diff
@@ -43,18 +43,89 @@ PyObject *CPyStr_GetItem(PyObject *str, CPyTagged index) {
     }
 }
 
-PyObject *CPyStr_Build(int len, ...) {
-    int i;
+// A simplification of _PyUnicode_JoinArray()
```

Comment:
Add `... from CPython X.Y.Z` to the end of the comment to make it clear from where it originally came from.